lib/class-wp-rest-widget-types-controller.php

```diff
@@ -399,6 +406,8 @@ public function get_item_schema() {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_widget_form( $request ) {
+		_deprecated_function( __METHOD__, '10.2.0' );
```

Comment:
Why deprecate this method?

Reply:
It is redundant now that `/wp/v2/widgets` has `rendered_form` and `?preview=1`.